blockstore/splitstore/splitstore.go

```diff
@@ -408,12 +455,12 @@ func (s *SplitStore) Put(ctx context.Context, blk blocks.Block) error {
 	s.debug.LogWrite(blk)
 
 	// critical section
-	if s.txnMarkSet != nil {
+	if s.txnMarkSet != nil && s.compactType == hot { // puts only touch hot store
```

Comment:
Do we also care about warmup here?

Reply:
We don't because warmup does not have a critical section